packages/nodes-base/nodes/Airtop/actions/window/takeScreenshot.operation.ts

```diff
@@ -7,12 +12,26 @@ import {
 } from '../../GenericFunctions';
 import { apiRequest } from '../../transport';
 
+export const description: INodeProperties[] = [
```

Comment:
Need to add `...takeScreenshot.description,` to `packages/nodes-base/nodes/Airtop/actions/window/Window.resource.ts`